archivers/libzzip/Portfile

```diff
@@ -47,31 +43,12 @@ post-destroot {
 
     xinstall -m 0644 {*}[glob -directory ${worksrcpath}/docs "*.{css,htm}"] \
         ${destroot}${docdir}/html
-
-    # We replace every soelim stub with a soft link, as ".so" groff directives
-    # don't work with gzip'd manpages.
-
-    foreach {f} [glob -directory ${destroot}${prefix}/share/man/man3 *.3] {
-        set channel [open ${f}]
-
-        if {[regexp {^\.so (.+)$} [read -nonewline ${channel}] _ target]} {
-            # This manpage is a soelim stub.
-            ln -sf [file tail ${target}] ${f}
-        }
-
-        close ${channel}
-    }
 }
 
 variant sdl description {Enable SDL support} {
     depends_lib-append      port:libsdl
 
-    configure.args-append   --enable-sdl
```

Comment:
You probably can't *just* remove this flag. The port needs to do *something* to ensure that libsdl will not be used if the variant is not selected, even if libsdl is already installed.